config/default.js

```diff
@@ -1,15 +1,16 @@
 var apiPort = process.env.NODE_PORT || 3000; // eslint-disable-line no-var
-var apiUrl = 'http://localhost:' + apiPort; // eslint-disable-line no-var
-var frontendUrl = 'http://localhost:8080'; // eslint-disable-line no-var
+var apiUrl = `http://localhost:${apiPort}`; // eslint-disable-line no-var
+const frontendUrl = 'http://localhost:8080'; // eslint-disable-line no-var
+const allowUrls = [frontendUrl, apiUrl]; // eslint-disable-line no-var
```

Comment:
As you are converting the old `var` to `const`, can you apply it to `apiUrl` too and remove the unnecessary eslint comments ?